cmd/state-installer/installer.go

```diff
@@ -154,3 +151,17 @@ func install(installPath string, cfg *config.Instance, out output.Outputer) erro
 
 	return nil
 }
+
+func mustOutputer() output.Outputer {
+	// init outputer
+	out, err := output.New("plain", &output.Config{
+		OutWriter:   os.Stdout,
+		ErrWriter:   os.Stderr,
+		Colored:     true,
+		Interactive: false,
+	})
+	if err != nil {
+		logging.Error("This shouldn't happen.  Err should never be != nil.")
+	}
+	return out
+}
```

Comment:
This makes unnecessary assumptions about how `output` works. If `output.New` errors then the program errors.